Claude and DetectBench prompt.

datvo06 opened the pull request:
This replicates [DetectBench Paper](https://arxiv.org/abs/2406.12641) on mysteries

datvo06 committed:
Adding DetectBench

datvo06 committed:
Initial trial with ollama deepseek r1

datvo06 committed:
Add Claude utils

datvo06 committed:
minor

datvo06 committed:
Pretty print the log

datvo06 committed:
Improve prompt

datvo06 committed:
Better logging

datvo06 committed:
Allow continuing after paused/suspended

datvo06 committed:
Partial result

datvo06 committed:
Full run results

namin commented on main_detect_bench_prompt.py:
```diff
@@ -0,0 +1,205 @@
+from claude import generate as claude_generate
+import tqdm
+
+# MODEL='deepseek-r1'
```
delete instead of comment out

datvo06 replied:
Thanks! I have deleted these lines in the latest commit.

namin commented on .gitmodules:
```diff
@@ -4,3 +4,6 @@
 [submodule "MuSR"]
 	path = MuSR
 	url = https://github.com/Zayne-sprague/MuSR.git
+[submodule "DetectBench"]
```
why do you add this as a submodule? do you use the repo? couldn't find where at a glance

datvo06 replied:
I don't use the repo yet, the prompt in the main script is inspired by it and the dataset is similar so I put it there for completeness (but might be redundant)
.

datvo06 committed:
Remove redundant line

namin commented:
It's definitely redundant to add the submodule if it is entirely unusused. How about removing that and adding a highlight in the README with a link to the repo and your results?


Thanks!

datvo06 committed:
Highlight DetectBench in README and remove the module

datvo06 committed:
Update gitmodules

datvo06 commented:
> It's definitely redundant to add the submodule if it is entirely unusused. How about removing that and adding a highlight in the README with a link to the repo and your results?
> 
> Thanks!

> It's definitely redundant to add the submodule if it is entirely unusused. How about removing that and adding a highlight in the README with a link to the repo and your results?
> 
> Thanks!

Thanks! I updated the README to highlight the DetectBench paper and added a link to the log.

namin commented:
Thanks! Please merge yourself when you’re ready.
